web/package.json

```diff
@@ -29,7 +29,6 @@
     "@types/react-redux": "^7.1.16",
     "@types/react-router-dom": "^5.3.0",
     "@types/webpack-env": "^1.15.1",
-    "@typescript-eslint/parser": "^5.0.0",
```

Comment:
Why this change?

Reply:
because https://github.com/bcgov/wps/pull/1442#discussion_r742220103

Reply:
Please fix with `git checkout origin/main -- path/to/package.json`